plugins/cloudservices/plugin.js

```diff
@@ -75,22 +75,55 @@
 		},
 
 		beforeInit: function( editor ) {
+			var tokenUrl = editor.config.cloudServices_tokenUrl,
+				tokenFetcher = {
+					token: null,
+
+					// Allow external code (tests) to affect token refresh interval if needed.
+					REFRESH_INTERVAL: editor.CLOUD_SERVICES_TOKEN_INTERVAL || 3600000,
+
+					refreshToken: function() {
+						CKEDITOR.ajax.load( tokenUrl, function( token ) {
+							if ( token ) {
```

Comment:
Maybe we should introduce some basic error handling? E.g. logging info to the console to inform the dev.

Reply:
This is already in place. If no token is returned dev will get `cloudservices-no-token` log. This behavior is covered by unit tests.